include/circt/Dialect/FIRRTL/Namespace.h

```diff
@@ -51,15 +51,16 @@ struct ModuleNamespace : public Namespace {
 
   /// Populate the namespace with the ports of a module-like operation.
   void addPorts(FModuleLike module) {
-    for (auto portSymbol : module.getPortSymbolsAttr().getAsRange<StringAttr>())
-      if (!portSymbol.getValue().empty())
-        nextIndex.insert({portSymbol.getValue(), 0});
+    for (auto portSymbol : module.getPortSymbolsAttr())
+      if (portSymbol)
+        nextIndex.insert(
+            {portSymbol.cast<InnerSymAttr>().getSymName().getValue(), 0});
```

Comment:
Agreed, I'm a bit concerned about `getSymName` because if it returns null for multiple per-field symbols, which might leads confusion in the future. (though we can fix later).